projects/ngx-datatable/src/lib/components/body/body-group-header.directive.ts

```diff
@@ -18,13 +18,13 @@ export class DatatableGroupHeaderDirective<TRow extends Row = any> {
   @Input() checkboxable = false;
 
   @Input('template')
-  _templateInput?: TemplateRef<GroupContext<TRow>>;
+  templateInputInternal?: TemplateRef<GroupContext<TRow>>;
```

Comment:
This is something I never liked in our rules. `_` should be required for internal fields. Lets disucuss before we also worsen the code here.